source/includes/APIReference/EffectiveSettings/Examples/_object.md.erb

```diff
@@ -143,7 +143,9 @@
   },
   "quickbooks": {
     "settings": {
-      "installed": 0
+      "installed": "1",
```

Comment:
Are you sure the value for _installed_ should be in quotes?  I'd expect it not to be.